src/main/java/org/apache/ibatis/executor/BatchExecutor.java

```diff
@@ -149,9 +150,6 @@ public int doUpdate(MappedStatement ms, Object parameterObject) throws SQLExcept
       }
       return results;
     } finally {
-      for (Statement stmt : statementList) {
```

Comment:
In this changes, If an exception occurred during execution statement, I think there is a case that statement does not close.

Reply:
Thank you for review - you are right. I will improve the code.